spec.html

```diff
@@ -29766,7 +29766,7 @@ <h1>String.prototype.repeat ( _count_ )</h1>
           1. Let _n_ be ? ToInteger(_count_).
           1. If _n_ &lt; 0, throw a *RangeError* exception.
           1. If _n_ is *+&infin;*, throw a *RangeError* exception.
-          1. If _n_ is 0, return the empty String.
+          1. If _n_ is *+0*, return the empty String.
```

Comment:
```suggestion
          1. If _n_ is *±0*, return the empty String.
```
Because `_count_` might be `-0`.